config/initializers/constants.rb

```diff
@@ -0,0 +1,22 @@
+IMAGE_FILE_EXTENSIONS = ['jpg', 'JPG', 'jpeg', 'JPEG', 'png', 'PNG']
+IMAGE_FILE_EXTENSIONS_PATTERN = /jpg|JPG|jpeg|JPEG|png|PNG/
+TIFF_FILE_EXTENSIONS_PATTERN = /tif|TIF|tiff|TIFF/
+
+INGESTOR_ALLOWLIST =  [
+  'title',
+  'identifier',
+  'description',
+  'restrict_scribes',
+  'physical_description',
+  'document_history',
+  'permission_description',
+  'location_of_composition',
+  'author',
+  'transcription_conventions',
+  'scribes_can_edit_titles',
+  'supports_translation',
+  'translation_instructions',
+  'pages_are_meaningful',
+  'document_set',
+  'slug'
+].freeze
```

Comment:
This should fix the warning messages we are getting. Moving forward if we have global constants we should declare it here in constants.rb

Reply:
Nice.  Any opinion on how to handle global enums?